developer-support/release-notes/portal.mdx

```diff
@@ -16,6 +16,82 @@ Our minor releases are supported until our next minor comes out.
+
+In this release, we have added an integrated GraphQL Playground to the Developer Portal, upgraded the Tyk Portal to Golang 1.24, and fixed a high-priority CVE.
+For a comprehensive list of changes, please refer to the detailed [changelog](#Changelog-v1.15.0) below.
+
+#### Breaking Changes
+
+There are no breaking changes in this release.
+
+#### Deprecations
+
+There are no deprecations in this release.
+
+#### Upgrade instructions
+
+If you are on version 1.14.0 or an older version, we recommend upgrading to this release ASAP.
+
+To upgrade the Portal's theme, please follow the [upgrade instructions](/portal/customization/themes#upgrading-themes).
+
+#### Download
+- [Docker image v1.15.0](https://hub.docker.com/r/tykio/portal/tags?page=&page_size=&ordering=&name=v1.15.0)
+  - ```bash
+    docker pull tykio/portal:v1.15.0
+    ```
+
+- [The default theme package](https://github.com/TykTechnologies/portal-default-theme/releases/tag/1.15.0)
+
+#### Changelog
+<a id="Changelog-v1.15.0"></a>
+
+##### Added
+
+<AccordionGroup>
+
+<Accordion title={'Tyk Portal now runs on Golang 1.24'}>
+The Tyk Portal has been updated to Golang 1.24, improving security by staying current with the latest Go versions.
+
+</Accordion>
+
+<Accordion title={'GraphQL Playground'}>
+We've introduced a comprehensive GraphQL Playground experience in the Developer Portal, providing developers with a seamless way to explore, test, and interact with GraphQL APIs directly within the portal environment.
+
+**Key Highlights:**
+
+- **Interactive GraphQL Playground**: Developers can now explore schemas, run queries and mutations, and test API behavior without leaving the Live Portal.
+- **Schema Documentation**: Admins can upload GraphQL Schema Definition Language (SDL) files (.graphql, .graphqls, or JSON format) to provide rich documentation and enable full playground functionality.
+- **Flexible Schema Loading**: The playground loads the schema information when SDL files are provided during API product configuration, ensuring developers have access to comprehensive API documentation.
+- **Documentation-Only GraphQL Products**: Create standalone GraphQL documentation products by uploading SDL files without requiring live API access, perfect for pre-release documentation or educational purposes.
+
+This feature is available only for APIs explicitly marked as GraphQL, and does not affect existing REST API behavior.
+
+Learn more about GraphQL documentation and playground features in our [API Products documentation](/portal/api-products#GraphQL-Playground-in-the-Developer-Portal)
+
+</Accordion>
+
+</AccordionGroup>
+
+##### Security Fixes
+
+<AccordionGroup>
+
+<Accordion title={'High priority CVEs fixed'}>
+Fixed the following high-priority CVEs identified in the Tyk Portal, providing increased protection against security
+vulnerabilities:
+
+- <a href="https://nvd.nist.gov/vuln/detail/cve-2020-8911" target="_blank">CVE-2020-8911</a>
+</Accordion>
+
+</AccordionGroup>
```

Comment:
@JoanCamosTyk, also note the new structure for adding release note items. we no longer use `<ul></ul>` etc. The previous release includes some elements of this that need to be removed; it's an error in the migration script.